gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -705,6 +714,15 @@ public JsonWriter newJsonWriter(Writer writer) throws IOException {
   }
 
   /**
+   * Returns a new JSON writer configured for the settings on this Gson instance.
+   */
+  public JsonReader newJsonReader(Reader reader) {
```

Comment:
Parity with `newJsonWriter` and protection against future configuration options being added.